docs/0-Preparing/starting_side_projects.md

```diff
@@ -0,0 +1,93 @@
+
+- It's a chance everyone has to write code. Getting better at coding requires writing code!
+- It's a chance to build something from scratch
+    - You don't always get to do that at work
+    - You get to see the whole picture
+    - It's satisfying
+- You can learn and explore different technologies with little commitment
+- They can be fun and allow you to enjoy programming in a way that your job can't
+- They help get you employed
+- They really help get you employed when you don't have industry experience
+- They open up more doors even when you have experience
+
+### Important notice
+
+While side projects have a lot of benefits, it doesn't mean that they should take up all your free time. **PLEASE PLEASE please don't spend all your time on coding projects unless that's just how you like to spend your time** you don't have to do side projects to be successful even though it often is talked about as if you do.
+
+### Side projects from the perspective of the employer
+
+Among other misconceptions, side projects don't have to be novel, amazing, or even useful. For internships and junior positions especially, all that an employer cares about is *interest and willingness to learn*. Here are different "signals" that a side project can convey, sorted roughly by the amount of experience associated with them. **Any one of these can be interesting for an employer**!
+
+**You can learn a technology on your own**
+
+- *Example*: You're reading through a book on learning language X, and applying what you learned by coding a few Project Euler problems.
+- *Why it's useful*: Most employers will hire you even if you don't know all the technologies that they use, and will expect you to learn some on the job. Here, you've established that you can do that.
+
+**You can learn a concept on your own**
+
+- *Example*: You got interested in Big Data and read some blog posts about how work can be split and parallelized to run on multiple machines.
+- *Why it's useful*: Employers sometimes have hard problems to solve and like to know that they can give you a bunch of readings on a new concept.
+
+**You can come up with project ideas**
+
+- *Example*: You noticed that the school newspaper club manually tallies up survey results into different demographics. You write a script that automates this work for them and generates pretty graphs.
+- *Why it's useful*: Employers like to see that you can take initiative.
+
+**You can complete a project on your own**
+
+- *Example*: You make a website to create and send reminders like your parent's birthday. It has user authentication, and a bunch of other thing so other people can use it.
+- *Why it's useful*: It's good to complete a project at some point, as you can learn different things at different stages of a project.
+
+As you can see, the first few don't require you to do anything extraordinary, and can be done even if you've only been programming for two weeks! [See this relevant XKCD](https://xkcd.com/519/).
```

Comment:
maybe take out "as you can see"

if I was really new to programming it could be hard for me to know which things take more time (e.g. are project euler questions really hard? I might not know the first few are quite basic)